app/screens/edit_post/edit_post_input/index.tsx

```diff
@@ -104,6 +111,14 @@ const EditPostInput = forwardRef<EditPostInputRef, PostInputProps>(({
                 onFocus={onFocus}
                 onBlur={onBlur}
             />
+            <Uploads
+                channelId={post.channelId}
+                currentUserId={post.userId}
+                files={postFiles}
+                uploadFileError={undefined} // TODO: Add upload file error
```

Comment:
Just verifying this todo goes into a different PR.

Reply:
Yup, in coming subsequent PR's.